packages/components/src/components/contract-card/contract-card-items/contract-card-body.jsx

```diff
@@ -26,15 +27,17 @@ const ContractCardBody = ({
     has_progress_slider,
     is_mobile,
     is_multiplier,
-    is_sold,
     is_turbos,
+    // is_positions,
```

Comment:
```suggestion
    is_positions,
```
@hamza-deriv  is_positions is used for determining if the card is now open in Recent positions (in Dtrader) or in Contract details. please uncomment it

Reply:
done